app/views/admin/events/_form.html.erb

```diff
@@ -1,53 +1,58 @@
 <section>
-<h1>Basic Info</h1>
-<%= form.text_field :name %>
-<%= form.check_box :active %>
-<%= form.select(:event_mode, Event.event_modes.keys.map {|event_mode| [event_mode.titleize, event_mode]}, {}, {:onChange => "checkEventMode();"}) %>
-<% if @event.eventbrite_token.nil? || @event.eventbrite_id.nil? %>
-<div id="missing_eb_warning" class="alert alert-warning" role="alert" style="display: hidden">
-  <p>
-    There is no EventBrite data for this event, so cannot be updated, please contact <%= mail_to "geeks@londondecom.org", "geeks@londondecom.org", subject: "Missing EventBrite Data for Event ID #{@event.id}" %> to get the data added.
-  </p>
-</div>
-<% end %>
+  <h1>Basic Info</h1>
+  <%= form.text_field :name %>
+  <%= form.check_box :active %>
+  <%= form.select(:event_mode, Event.event_modes.keys.map {|event_mode| [event_mode.titleize, event_mode]}, {}, {:onChange => "checkEventMode();"}) %>
+  <% if @event.eventbrite_token.nil? || @event.eventbrite_id.nil? %>
+    <div id="missing_eb_warning" class="alert alert-warning" role="alert" style="display: none">
```

Comment:
why did this change?

Reply:
Because display dones't have the property `hidden` it does have `none`

Reply:
display can have the property hidden: https://developer.mozilla.org/en-US/docs/Web/CSS/visibility

Reply:
Trying `display: hidden` in the browser results in `Invalid property value`, however if we changed the line to be `style="visibility: hidden"` then we could use `hidden`

Reply:
oh! I see. My mistake.